src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -106,8 +115,10 @@ class IndexLogEntryTest extends SparkFunSuite {
       Seq(
         Relation(
           Seq("rootpath"),
-          Hdfs(Hdfs.Properties(
-            Content("", Seq(Content.Directory("", Seq("f1", "f2"), NoOpFingerprint()))))),
+          Hdfs(
+            Hdfs.Properties(Content(
+              "",
+              Seq(Content.Directory("", Seq("f1", "f2").map(toFileInfo), NoOpFingerprint()))))),
```

Comment:
Can you create `FileInfo` by hand here? `Seq(FileInfo(...), )` with non zero values?

Reply:
To me, just handcrafting the `FileInfo` isn't too bad.